capstone/src/main/java/com/google/sps/servlets/Constants.java

```diff
@@ -55,4 +55,5 @@ public Constants() {}
   public static final String TIMEZONE_OFFSET = ":00.000-07:00";
   public static final String TIMEZONE_PROP = "timezone";
   public static final String EMAIL_PATH = "/emailTemplates/";
+  public static final String REQUEST_PROP = "requests";
```

Comment:
BTW, not exclusive to this PR but the Constants file is getting a bit long and unstructured. I know it's your last (short) week and you might not have time to do this, but can you file a GitHub Issue to refactor this class and organize it better. Then you can link a TODO comment at the top of the class for tracking purposes.